tests/core/snap-auto-mount/task.yaml

```diff
@@ -1,7 +1,8 @@
 summary: Check that `snap auto-import` works as expected
 
-# TODO:UC20: enable for UC20
-systems: [ubuntu-core-1*-64]
+# TODO:UC20: fix for UC20 and enable, currently fails because mkfs.vfat is not
```

Comment:
I need to check if it works on arm device.

Reply:
I'll remove these for now and just run on all systems except arm with:
```yaml
systems: [ubuntu-core-*-64]
```

Reply:
err actually for this one, I will leave it disabled for UC20, so use this:

```
systems: [ubuntu-core-1*-64]
```